graphics/opencv4/Portfile

```diff
@@ -59,86 +62,52 @@ proc move_gh_repo {repo_dir_patt new_name} {
     }
 }
 
-post-extract {
-    move_gh_repo ${github.author}-${github.project}-* ${distname}
-}
-
 compiler.cxx_standard 2011
 
-compiler.blacklist-append \
-                    *gcc* \
-                    {clang < 900} \
-                    {macports-clang-3.[0-9]} \
-                    {macports-clang-[4-6].0}
+compiler.blacklist-append *gcc* {clang < 900} {macports-clang-3.[0-9]} {macports-clang-[4-6].0}
 
 depends_build-append \
                     port:pkgconfig
 
-depends_lib-append  \
-                    port:ffmpeg \
-                    port:zlib \
+depends_lib-append  port:ade \
+                    port:ilmbase \
+                    port:jasper \
+                    port:jpeg \
                     port:libpng \
-                    path:include/turbojpeg.h:libjpeg-turbo \
```

Comment:
You have reverted an important change that switches macports to libjpeg-turbo. This needs to be restored.

Reply:
Thanks, done.

Reply:
@mf2k This appears to create a dependency hell issue with port `jpeg`. Please advise.

```bash
--->  Computing dependencies for fltk
Error: Can't install jpeg because conflicting ports are active: libjpeg-turbo
--->  Computing dependencies for tiff
Error: Can't install jpeg because conflicting ports are active: libjpeg-turbo
--->  Computing dependencies for libgeotiff
Error: Can't install jpeg because conflicting ports are active: libjpeg-turbo
--->  Computing dependencies for webp
Error: Can't install jpeg because conflicting ports are active: libjpeg-turbo
--->  Computing dependencies for jasper
Error: Can't install jpeg because conflicting ports are active: libjpeg-turbo
--->  Computing dependencies for lcms
Error: Can't install jpeg because conflicting ports are active: libjpeg-turbo
--->  Computing dependencies for netpbm
Error: Can't install jpeg because conflicting ports are active: libjpeg-turbo
```

Reply:
That's because you also reverted the change that removes the dependency on jpeg.

Reply:
Doh! Thanks.